solutions/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from nanoid import generate
+
+def generate_nanoid():
+    return generate(size=12)
+
+# Create your models here.
+
+class SolutionHistory(models.Model):
+    id = models.CharField(
+    primary_key=True,
+    default=generate_nanoid,
+    editable=False,
+    unique=True,
+    max_length=21,
+    )
+    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE)
+    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+    solved_at = models.DateTimeField(null=True, blank=True)
+    is_solved = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.title
```

Comment:
❗ SolutionHistory 모델은 title 필드를 갖지 않으므로 해당 코드는 오류가 발생합니다. 만약 Mission 모델의 title 필드를 참조하고 싶으셨던 것이라면
```suggestion
    def __str__(self):
        return self.mission.title
```
이렇게 작성해야 합니다.

💊 그런데 제가 생각하기에 한 개의 Mission에 여러 개의 SolutionHistory이 존재할 수 있으므로 `mission.title`로만 표시하면 admin 사이트에서 디버깅하기에 불편할 성싶습니다.
```suggestion
    def __str__(self):
        return f"{self.user} {self.mission.title} {self.created_at}"
```
이런 형식은 어떨까요?